src/dlgTriggerEditor.cpp

```diff
@@ -1805,8 +1728,6 @@ void dlgTriggerEditor::slot_deleteTimer()
     if( ! pItem ) return;
     QTreeWidgetItem * pParent = pItem->parent();
 
-    if( ! pItem ) return;
```

Comment:
There may be other redundant tests like this in this class: I thought I had spotted and removed a few recently where we had this sort of test then _later_ on in the same method a test for the opposite state _even though the variable was not modified in between_ which would thus have it's branch always be executed.  Perhaps they do not trigger a Coverity warning in the same way (or are all gone now)...